tests/migrations/test_graph.py

```diff
@@ -178,28 +176,25 @@ def test_graph_recursive(self):
         backwards_plan = graph.backwards_plan(root)
         self.assertEqual(expected[::-1], backwards_plan)
 
-    def test_graph_iterative(self):
+    def test_iterative_dfs_complexity(self):
```

Comment:
I think this could do with a small docstring (or a comment by `expected`). 

It takes an age to run on the old implementation. 45 minutes in, with 23GB of RAM used, I killed it. 
I wonder if a smaller `n` would make the same point?

Reply:
I reduced `n` to 50 and added a docstring.